paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -109,8 +110,8 @@ class Paparazzi @JvmOverloads constructor(
   private val contentRoot = """
```

Comment:
I'd leave a comment about what this is and that it is dynamic.

Reply:
Fixed in [0d9a21a](https://github.com/cashapp/paparazzi/pull/497/commits/0d9a21ae563fec566ec2f93783fb61c7b9699eaa)